openedx/features/edly/signals.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from lms.djangoapps.instructor.enrollment import get_email_params
+from openedx.features.edly.tasks import send_course_enrollment_mail
+from student.models import CourseEnrollment
+
+
+@receiver(post_save, sender=CourseEnrollment)
+def handle_user_enroll(sender, instance, **kwargs):
+    """
+
+    :param sender: Model from which we received signal.
+    :param instance: Instance of model which has been created or updated
+    :param kwargs: Remaining parts of signal.
+    """
+
+    # This if condition check's for if the Enrollment signal is created/updated using LMS or not.
+    # We are handling only lms site.
+
+    if settings.ROOT_URLCONF == 'lms.urls':
+        user_fullname = instance.user.profile.name
+        user_email = instance.user.email
+        email_params = get_email_params(instance.course, True, secure=False)
+        email_params['email_address'] = user_email
+        email_params['full_name'] = user_fullname
+        email_params['enroll_by_self'] = True
+        if instance.is_active:
+            email_params['message_type'] = 'enrolled_enroll'
+        elif not instance.is_active and not kwargs['created']:
+            email_params['message_type'] = 'enrolled_unenroll'
+        else:
+            return
+        send_course_enrollment_mail.delay(user_email, email_params)
```

Comment:
I got this on enrolling in one of the course.



```

------- EMAIL -------
To: edx@example.com
From: 
Subject: You have been enrolled in hassan_demo_2
Body:
Dear 

You have been enrolled in hassan_demo_2 at localhost:18000 by a member of the course staff. This course will now appear on your localhost:18000 dashboard.

To start accessing course materials, please visit http://localhost:18000/courses/course-v1:ucp+CS1034+2014_T2/

----
This email was automatically sent from localhost:18000 to
-------  END  -------
```
Do you think that this is the right  Email content?
Kindly have a look at the last line of the content and `From` option too.

Reply:
That's a dummy email for debugging you can see the actually html email by going to your `src\ace_messages\*.log`

Reply:
I checked it but the link is not enclosed in the `anchor` tag it is in the `paragraph` tag.

![image](https://user-images.githubusercontent.com/42294172/67757736-2ec45780-fa5e-11e9-89f9-d833c048786f.png)

Reply:
![image](https://user-images.githubusercontent.com/23108499/67759428-36d1c680-fa61-11e9-9ed3-6f9ea5dff1a6.png)

Reply:
but in my case I tested it again and I am still not getting an `anchor` tag

Reply:
There is no anchor tag try to open this template in your browser don't render the ace message template.